src/v/config/configuration.cc

```diff
@@ -2569,6 +2569,36 @@ configuration::configuration()
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       std::nullopt,
       {.min = 1.0, .max = 100.0})
+  , cloud_storage_inventory_based_scrub_enabled(
+      *this,
+      "cloud_storage_inventory_based_scrub_enabled",
+      "Scrubber uses cloud storage inventory report to check if the required "
+      "objects exist in bucket.",
```

Comment:
```suggestion
      "objects exist in the bucket.",
```

Reply:
fixed in https://github.com/redpanda-data/redpanda/pull/21576